### DISCUSSION TITLE
请问IoTDB是如何区分实体（也就是设备的）

### DISCUSSION THREAD

**Discussion by SpikeFJ** [apache/iotdb#11394] [Q&A]
目前我们一个业务数据，之前的设计是实体（也就是设备）下面挂物理量

由于数据比较种类比价多，且不同种类间数据不具备业务联系，需要对不同的数据进行一定的分组划分。

所以就想在实体下再挂一层数据种类。

但是不知道IoTDB是如何知道哪一层级是实体的，叶子节点是物理量，倒数第二级就是实体吗？

**Comment by wanghui42** [comment]
是的，iotdb的设备是倒数第二层。设备（实体）的语义与现实中的设备不是同一个。建议根据您的查询来设置叶子节点。

**Reply by SpikeFJ** [reply]
我看到有一些专门针对设备的语句，如show/count devices ，但是没看到delete，请问是为什么呢?
